tools/ratbagd.py

```diff
@@ -289,10 +289,41 @@ def commit(self):
 class RatbagdProfile(_RatbagdDBus):
     """Represents a ratbagd profile."""
 
+    CAP_WRITABLE_NAME = 100
+
     def __init__(self, object_path):
         _RatbagdDBus.__init__(self, "Profile", object_path)
 
     @GObject.Property
+    def capabilities(self):
+        """The capabilities of this profile as an array. Capabilities not
+        present on the device are not in the list. Thus use e.g.
+
+        if RatbagdProfile.CAP_WRITABLE_NAME is in profile.capabilities:
+            do something
+        """
+        return self._get_dbus_property("Capabilities")
+
+    @GObject.property
+    def name(self):
+        """The name of the profile"""
+        return self._get_dbus_property("Name")
+
+    @name.setter
+    def name(self, name):
+        """Set the name of this profile.
+
+        @param name The new name, as string"""
+        if RatbagdProfile.CAP_WRITABLE_NAME not in self.capabilities:
```

Comment:
I had a chat with @whot and the summary is:
- ratbagd should be a thin wrapper for libratbag only and should not do semantic checks
- ratbagd assumes the client is doing the right thing, so it just accepts anything we provide to it
- libratbag during `commit()` will check for inconsistency (is the writable capability set, is the value within the range), and in case of an issue, it will raise a generic `please kindly resync the device, you screwed up` signal to ratbagd

This should make everybody happy ever after